baseband/mark4/payload.py

```diff
@@ -218,11 +221,11 @@ def __init__(self, words, header=None, nchan=1, bps=2, fanout=1):
             self._size = header.payloadsize
         self._dtype_word = MARK4_DTYPES[nchan * bps * fanout]
         self.fanout = fanout
+        sample_shape = self._sample_shape_cls(nchan)
```

Comment:
Doesn't vlbi_base/payload do this already?

Reply:
No, it's a `tuple` in `vlbi_base`.  If we declare it in `VLBIPayloadBase`, then any end-user who creates a new format will have to declare a proper `sample_shape`.  I recommend leaving this for when we decide whether to implement `self._payload_cls._sample_shape_cls`.

Reply:
I don't think it is true any end user would have to declare anything!